docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md

```diff
@@ -61,19 +61,19 @@ If you enable remote debugging for a cloud service, it doesn't exhibit degraded
 
 ### To attach the debugger to a cloud service in Azure
 
-1. In Server Explorer, expand the node for your cloud service.
+1. In **Server Explorer**, expand the node for your cloud service.
 
 2. Open the shortcut menu for the role or role instance to which you want to attach, and then select **Attach Debugger**.
 
     If you debug a role, the Visual Studio debugger attaches to each instance of that role. The debugger will break on a breakpoint for the first role instance that runs that line of code and meets any conditions of that breakpoint. If you debug an instance, the debugger attaches to only that instance and breaks on a breakpoint only when that specific instance runs that line of code and meets the breakpoint's conditions.
 
     ![Attach Debugger](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC746719.gif)
 
-3. After the debugger attaches to an instance, debug as usual. The debugger automatically attaches to the appropriate host process for your role. Depending on what the role is, the debugger attaches to w3wp.exe, WaWorkerHost.exe, or WaIISHost.exe. To verify the process to which the debugger is attached, expand the instance node in Server Explorer. See [Azure Role Architecture](https://blogs.msdn.microsoft.com/kwill/2011/05/05/windows-azure-role-architecture/) for more information about Azure processes.
+3. After the debugger attaches to an instance, debug as usual. The debugger automatically attaches to the appropriate host process for your role. Depending on what the role is, the debugger attaches to w3wp.exe, WaWorkerHost.exe, or WaIISHost.exe. To verify the process to which the debugger is attached, expand the instance node in **Server Explorer**. See [Azure Role Architecture](https://blogs.msdn.microsoft.com/kwill/2011/05/05/windows-azure-role-architecture/) for more information about Azure processes.
 
     ![Select code type dialog box](./media/vs-azure-tools-debug-cloud-services-virtual-machines/IC718346.png)
 
-4. To identify the processes to which the debugger is attached, open the Processes dialog box by, on the menu bar, choosing Debug, Windows, Processes. (Keyboard: Ctrl+Alt+Z) To detach a specific process, open its shortcut menu, and then select **Detach Process**. Or, locate the instance node in Server Explorer, find the process, open its shortcut menu, and then select **Detach Process**.
+4. To identify the processes to which the debugger is attached, go to the **Processes** dialog box by, on the menu bar, selecting **Debug**, **Windows**, **Processes**. (Keyboard: Ctrl+Alt+Z) To detach a specific process, open its shortcut menu, and then select **Detach Process**. Or, locate the instance node in **Server Explorer**, find the process, open its shortcut menu, and then select **Detach Process**.
```

Comment:
I would do this:

_To identify the processes that are attached to the debugger, on the menu bar, select **Debug** > **Windows** > **Processes**, and then open the **Processes** dialog box. (Keyboard: Ctrl+Alt+Z) To detach a specific process, open its shortcut menu, and then select **Detach Process**. Or, locate the instance node in Server Explorer, find the process, open its shortcut menu, and then select **Detach Process**._

Reply:
Hi, there isn't a contextual screenshot or an environment for me to be sure about the relations among the UI elements. I assume you have checked the actual UI. So I'll make the update as per your suggestion.